homeassistant/components/alexa/entities.py

```diff
@@ -320,6 +323,8 @@ def default_display_categories(self):
         device_class = self.entity.attributes.get(ATTR_DEVICE_CLASS)
         if device_class in (cover.DEVICE_CLASS_GARAGE, cover.DEVICE_CLASS_DOOR):
             return [DisplayCategory.DOOR]
+        elif device_class in (cover.DEVICE_CLASS_BLIND)
```

Comment:
Looking at the [Alexa documentation](https://developer.amazon.com/de-DE/docs/alexa/device-apis/alexa-discovery.html#display-categories) I think it would be a good idea to include `INTERIOR_BLIND` as well.

Reply:
Thanks for the Feedback. Actually I saw already that there is a much bigger Pull Request which considers my already: [#29917](https://github.com/home-assistant/home-assistant/pull/29917)

I try to find out how to delete this pull request. This was my first one at all.

Reply:
Of course. Good catch, you are correct and this PR looks like a complete refactor to me.
Therefore, closing this PR is a good idea.